CHANGELOG.md

```diff
@@ -14,6 +14,10 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - OTLP .proto files from `open-telemetry/opentelemetry-proto` imported as a git submodule under `internal/opentelemetry-proto`.
    References to `github.com/open-telemetry/opentelemetry-proto` changed to `go.opentelemetry.io/otel/internal/opentelemetry-proto-gen`. (#948)
 
+### Changed
+
+- Non-nil structs will be marshalled using JSON rather than Sprintf. (#948)
```

Comment:
Might be good to add context of what struct.

```suggestion
- Non-nil value structs for key-value pairs will be marshalled using JSON rather than Sprintf. (#948)
```